### PR TITLE
Remove old checks - DO NOT MERGE

### PR DESCRIPTION
**Pull request by achiar99**
In this pr i removed the old tf v3 python checks and change the id of the new ones.
Also i fixed some bugs with the graph checks 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: checkov/common/checks_infra/solvers/connections_solvers/or_connection_solver.py [hunk 1]
```diff
@@ -34,6 +34,17 @@ def get_operation(self, graph_connector: DiGraph) -> Tuple[List[Dict[str, Any]],
                 failed.append(data["v"])
 
         passed.extend(passed_connections)
-        failed = [f for f in failed if f[CustomAttributes.ID] not in [p[CustomAttributes.ID] for p in passed]]
+        failed = OrConnectionSolver._filter_failed(failed, passed)
 
         return self.filter_results(passed, failed)
+
```
Comment (gruebel) [inline checkov/common/checks_infra/solvers/connections_solvers/or_connection_solver.py:40]: can you add the `@staticmethod` annotation here

**Review by gruebel — COMMENTED** [review]
looks good, added two comments

**Review by gruebel — APPROVED** [review]
nice 💯 merge when you want

**Comment by nimrodkor** [comment]
Linked to relevant issues, exciting stuff!

**Pull request merged**